README.md

```diff
@@ -12,6 +12,7 @@ This is Free and Open Source Software. You are welcome to use the code and set u
 
 ## Get involved!
 
+* Check our [Code of conduct](.docs/code_of_conduct.md)
```

Comment:
This must be a typo. It should be `./docs/code_of_conduct.md` in order to link to the correct document/page.

Reply:
@d-rita I fixed it with #3982